### DISCUSSION TITLE
关于使用主题的问题

### DISCUSSION THREAD

**Discussion by zycn0910** [YunYouJun/hexo-theme-starry#149] [Q&A 疑难杂症] [answered]
您好，我是hexo-yun主题的众多使用者之一，在我完成种种配置后，发现侧栏头像问题始终无法得到很好的解决，以及希望得到一份带有中文阐释的主题配置文件。

**Comment by YunYouJun** [comment, marked as answer]
事实上，主题配置文件的每个字段都在文档中有详尽的中文介绍。
hexo 的配置字段则请见 hexo 文档，不在主题文档范围中。

侧栏头像请参照示例网站中进行配置。

**Comment by zycn0910** [comment]
感谢您在百忙之中能抽出空来查看我的留言并进行回复，

我在查看您提供的文档之后尝试着配置我下载的主题目录下的文件，具体代码为`favicon: /yun.svg`（我直接更改了源文件，保留了原本的文件名）

在此配置之后，我直接反手进行了`hexo g`，但是访问博客时头像还是无法正常显示，具体表现为：

1. PC端显示为无法打开的图片图标（好像加载超时）
2. 移动端显示为空白

据我猜测是因为主题文件的图片加载使用了jsdelivr，但是我的库无法通过jsdelivr正常访问，因为我在`hexo s`测试时头像是可以正常显示的。

如果我的猜测是正确的，请您告诉我如何关闭jsdelivr。

如果我的猜测是错误的，希望您可以给予我些许建议。

如果您想进一步查看我的源文件，[请点击此处](https://github.com/zycn0910/zycn0910.github.io)

**Reply by YunYouJun** [reply]
头像并不是 favicon 而是 avatar.url。
详情见：https://yun.yunyoujun.cn/guide/config.html#%E5%A4%B4%E5%83%8F

**Comment by zycn0910** [comment]
我尝试配置主题目录下的文件之后，还是出现此问题
avatar:
  enable: true
  url: /yun.png
这次我还是直接替换的源文件。

**Reply by YunYouJun** [reply]
你这缩进不对啊……   以及不要替换源文件，自己放自己的文件夹里。

**Comment by zycn0910** [comment]
非常抱歉，出现此项问题的原因是我自己的失误（文件编码错误），再次感谢您的耐心回复。

除此之外我还想了解一下网站的留言板配置，因为本人比较菜，只会使用git talk，但很不巧的又没配置。我查看文档上的配置有些摸不着头脑，希望您能提供些建议。

关于我为何直接配置主题根目录文件，因为我按照文档上新建_data/yun.yml后`hexo g`出现各种错误。

**Reply by YunYouJun** [reply]
可参见 [第三方评论系统之我见](https://www.yunyoujun.cn/share/third-party-comment-system/)。

很遗憾，我并没有发现一款真正百无挑剔的评论系统。
